src/server/dfly_main.cc

```diff
@@ -609,6 +610,16 @@ void sigill_hdlr(int signo) {
   exit(1);
 }
 
+void PrintBasicUsageInfo() {
+  std::cout << "* Logs will be written to the following paths (in this order): \n";
```

Comment:
How about "Logs will be written to the first available of the following paths"?
Or maybe "Logs will try to use the following paths, in this order:"

BTW how many lines does this print on your machine?

Reply:
It prints two lines :)

Reply:
Because there could be runtime issues that will prevent glog from doing that, like permission issues, missing directory, empty disk, etc

Reply:
ohhh that makes sense :+1:

Reply:
Also my output is: 

```
/tmp/dragonfly.*
./dragonfly.*
```